data-changes/location-codes.md

```diff
@@ -1,45 +1,55 @@
 # Location codes change
 
-This document provides details of a correction we have applied in 2021 to correctly represent location code of GeoNet strong motion (SM) stations.
-The change has been applied to data and metadata which have been corrected during 2021. 
+This document provides details of **Data/Metadata Legacy corrections** we have applied to correctly label the **location code** of **strong motion instruments** part of the GeoNet national network for periods of time between **2011 and 2020**.
```

Comment:
Suggestion:

"This document provides details of **Data/Metadata Alignment** that has been applied to correct the **location code** labels for a number of **strong motion stations** in the GeoNet national network. The correction in location codes primarily covers the time period betwwen **2011 to 2020** ". 

--note: we didn't really correct the data per se, we changed the filename of the data to align with the metadata

Reply:
added : "NB: This type of change in the archive only affects stream names: file headers only are modified
"